src/index.js

```diff
@@ -86,10 +90,15 @@ module.exports = new BaseKonnector(async function fetch(fields) {
       const isIsp = facture.lignes.some(
         val => !['06', '07'].includes(val.numeroLigne.substr(0, 2))
       )
-      const categories = []
-      // some bills are related to isp and phone
-      if (isIsp) categories.push('isp')
-      if (isMobile) categories.push('phone')
+      let qualification = Qualification.getByLabel('telecom_invoice')
+      // some bills are related to isp and phone, try a better qualification
+      if (isIsp) {
+        qualification = Qualification.getByLabel('isp_invoice')
+      }
+      if (isMobile) {
```

Comment:
Is it possible to have both `isIsp` and `isMobile`? If yes, that would fall in `phone_invoice`, while I assume it should be `telecom_invoice`?

Reply:
Switch to an exclusive precise label and switch back to telecom_invoice if we have multi lines (mobile and not) in the same bill.

Reply:
Changed